zkevm-circuits/src/evm_circuit/execution/error_precompile_failed.rs

```diff
@@ -48,6 +48,20 @@ impl<F: Field> ExecutionGadget<F> for ErrorPrecompileFailedGadget<F> {
         let is_staticcall =
             IsZeroGadget::construct(cb, "", opcode.expr() - OpcodeId::STATICCALL.expr());
 
+        // constrain op code
+        // NOTE: this precompile gadget is for dummy use at the moment, the real error handling for
```

Comment:
issue https://github.com/scroll-tech/zkevm-circuits/issues/673 is tracking this.